### PR TITLE
Miscellaneous fixes relevant to text2time, time, timeseries AIT module tutorials

### LINKED ISSUE #95: PassiveAggressiveClassifier C parameter in text/defaults.py
Must have $C>0$ for sklearn's PassiveAggressiveClassifier, but text/defaults.py includes C=0 as one of the default values to search over. See below, which shows the left bound is open:
image

### LINKED ISSUE #96: AdaBoostClassifier algorithm parameter
According to scikit-learn's documentation, setting `algorithm="SAMME.R"` is deprecated and will be removed in future versions. Should be changed to `"SAMME"` to prevent breaking in the future.
https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.AdaBoostClassifier.html

### LINKED ISSUE #97: Known bug in scikit-learn's f1_score metric
Related to #29734 (https://github.com/scikit-learn/scikit-learn/issues/29734).

Scikit-learn's documentation states that the `pos_label` parameter is ignored in multiclass problems, but it isn't actually ignored. Since it defaults to `pos_label=1`, which isn't a label found in the dataset, the scoring fails every time, giving `np.nan`.

Setting `pos_label=None` explicitly would resolve the problem and have `f1_score` work as intended.

### LINKED ISSUE #98: Deep copy needed before each scaling in predict_subset() method of pvops/timeseries/models/AIT.py
Currently, when new standardized columns are created, the original columns are also standardized. This results in the R2 and logRMSE being calculated between the predicted unscaled energy and the true scaled energy, suggesting (incorrectly) that the model performs poorly.

There is a deep copy at the start of the method, but it doesn't do anything. The deep copy needs to happen inside the for-loop before each standardization is applied.

### LINKED ISSUE #99: FutureWarning when downcasting with pandas replace() function in AIT tutorial
See: https://github.com/pandas-dev/pandas/issues/57734

In tutorials/tutorial_AIT_timeseries.ipynb, the dc capacity for each site is added to `filtered_prod_data` using `replace()` to map between the site id and the site capacity. However, this implicitly downcasts the column from `object` dtype to `int64`. This behavior is deprecated and will be removed in future versions of pandas.

The recommended approach based on the warning is to use `pd.option_context(future.no_silent_downcasting, True)` and then cast the column's dtype manually. However, in later releases of pandas once this behavior becomes standard, this context manager will itself become deprecated according to: https://pandas.pydata.org/docs/reference/api/pandas.describe_option.html

A better approach may be doing the replacement more explicitly via a masking approach, which is still simple and readable and avoids further reworks of this code with updates to pandas.

### PR DESCRIPTION
**Pull request by agmoore4**
Resolves #95, resolves #96, resolves #97, resolves #98, resolves #99

### CODE REVIEW

**Review by kbonney — APPROVED** [review]
Looks great!
